crates/precompile/Cargo.toml

```diff
@@ -15,7 +15,7 @@ reth-revm.workspace = true
 alloy-primitives.workspace = true
 
 p256 = { version = "0.13.2", features = ["ecdsa"] }
-
+revm-precompile = { git = "https://github.com/bluealloy/revm.git", features = ["secp256r1"] }
```

Comment:
isn't this available on revm-precompile 12.0.0 ?

Reply:
not sure of that

Reply:
this can't use git deps

Reply:
replaced with crates version